Packet++/src/TcpReassembly.cpp

```diff
@@ -159,6 +160,7 @@ TcpReassembly::ReassemblyStatus TcpReassembly::reassemblePacket(Packet& tcpData)
 		}
 	}
 
+	timeval timestampOfTheRecievedPacket = timespecToTimeval(tcpData.getRawPacket()->getPacketTimeStamp());

```

Comment:
we already fetch the packet timestamp if the connection isn't new [here](https://github.com/seladb/PcapPlusPlus/blob/d8d702acc99bf8bbf1531f00a3162c6b11a3cd2c/Packet%2B%2B/src/TcpReassembly.cpp#L147). Maybe we should just move it to after [this line](https://github.com/seladb/PcapPlusPlus/blob/d8d702acc99bf8bbf1531f00a3162c6b11a3cd2c/Packet%2B%2B/src/TcpReassembly.cpp#L113) so we don't call the same method twice?

Reply:
Should i replace the call in [this](https://github.com/seladb/PcapPlusPlus/blob/d8d702acc99bf8bbf1531f00a3162c6b11a3cd2c/Packet%2B%2B/src/TcpReassembly.cpp#L128) line too?

Reply:
Yes, I think so. Thanks for pointing this out, I missed it... :)